components/region_cache_memory_engine/src/background.rs

```diff
@@ -340,8 +343,9 @@ impl BackgroundRunnerCore {
 
         let mut iter = write_cf_handle.iterator();
         let guard = &epoch::pin();
-        iter.seek_to_first(guard);
-        while iter.valid() {
+        let (start_key, end_key) = encode_key_for_eviction(range);
```

Comment:
`eoncode_key_for_eviction` is not a proper name here, here you just want to reuse the implementation of this function, but the semantic is not correct.

Reply:
Better to rename `encode_key_for_eviction` to `encode_range_keys` or similar name. The name `encode_key_for_eviction` is not well abstracted, and it is too specific.

Reply:
In my another PR, I have change it to `encode_key_for_boundary_with_mvcc`. Also, there's another method called `encode_key_for_boundary_without_mvcc` in that PR.